sdk/cosmos/azure-cosmos-spark_3_2-12/Samples/DatabricksLiveContainerMigration/README.md

```diff
@@ -34,13 +34,34 @@ Select the latest Azure Databricks runtime version which supports Spark 3.0 or h
 
     ![image](./media/import-notebooks.jpg)
 
-## Attach cluster and Run
+## Attach cluster and Run - multiple containers migration
+
+* In the `CosmosDBLiveParallelContainerMigration` notebook, attach the cluster you created earlier:
+
+    ![image](./media/attach-cluster-p.jpg)
+
+* In the `cosmosDBLiveMigrationList.csv` csv, replace the references to Cosmos DB account URI, key, region, source database/container, target database/container/provisioned throughput, and target container partition key values as approprate:
+
+    ![image](./media/metadata-p.jpg)
+
+* Upload `cosmosDBLiveMigrationList.csv` csv to dbfs:
+    ![image](./media/upload-dbfs.jpg)
+
+* Change the number of containers you want to be migrated in parallel:
+
+    ![image](./media/notebooks-in-p.jpg)
+
+* As long as everything has been configured correctly, all cells should run in sequence. The final cell is executing `CosmosDBLiveContainerMigration` notebook, which is detailed below:
+
+    ![image](./media/live-migration-p.jpg)
+
+## Attach cluster and Run - single container migration
 
 * In the `CosmosDBLiveContainerMigration` notebook, attach the cluster you created earlier:
 
     ![image](./media/attach-cluster.jpg)
 
-* In the `CosmosDBLiveContainerMigration` notebook, replace the references to Cosmos DB account URI, key, source database/container, target database/container, and target container partition key values as approprate that are contained within each cell (see comments in each cell for guidance). Then you are ready to click "Run all":
+* In the `CosmosDBLiveContainerMigration` notebook, replace the references to Cosmos DB account URI, key, region, source database/container, target database/container/provisioned throughput, and target container partition key values as approprate that are contained within each cell (see comments in each cell for guidance). Then you are ready to click "Run all":
```

Comment:
In the image of the csv file, I suggest we have some real values. In particular, make it clear that partition key needs forward "/" slash (new users often will get this wrong). Failure caused by this will not be transparent in the single container migration notebook.